Scripts/Src/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.IO;
+
+namespace BootstrapScript
+{
+    class Program
+    {
+        static int Main(string[] args)
+        {
+            string appName = args[0];
+            Console.WriteLine("Setting up bootstrap for " + appName);
+            string bootstrapRootDir = "";
+            string bootstrapName = "NetCoreBootstrap";
+            foreach (var dir in Directory.GetCurrentDirectory().Split('/'))
+            {
+                bootstrapRootDir += dir + "/";
+                if (dir == appName) break;
+            }
+            Console.WriteLine("Replacing " + bootstrapName + " to " + appName + " in " + bootstrapRootDir);
+            string contents = "";
+            foreach (string file in Directory.EnumerateFiles(bootstrapRootDir, "*", SearchOption.AllDirectories))
+            {
+                if (file.Contains("bootstrap-script")) continue;

```

Comment:
You could use Linq's filter function to eliminate this case from the iteration and avoid using `continue`